pkg/driver_atwinc15x0/Kconfig

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config PACKAGE_DRIVER_ATWINC15X0
+    bool "ATWINC15x0 WiFi driver package"
+    depends on TEST_KCONFIG
+    depends on HAS_PERIPH_SPI
+    select MODULE_PERIPH_SPI
+    depends on HAS_PERIPH_GPIO_IRQ
+    select MODULE_PERIPH_GPIO_IRQ
```

Comment:
Is it possible to have MODULE_PERIPH_GPIO_IRQ and no MODULE_PERIPH_GPIO? 

If not I would probably say this should be resolved earlier?

I can add it in if we say it is better to be verbose though.

Also should I group based on `depends on` and `select` or by the modules?

Reply:
I will add it for now.  Refactoring would be done later anyways.

Reply:
You can't activate MODULE_PERIPH_GPIO_IRQ without MODULE_PERIPH_GPIO. In the makefile we don't see this because it's done automatically somewhere else:
https://github.com/RIOT-OS/RIOT/blob/6ce9db6df8b44fc8253241a10c64108a138a29d6/drivers/Makefile.dep#L199-L202